specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/ManagedInstanceTdeCertificates.json

```diff
@@ -0,0 +1,245 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-10-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/tdeCertificates": {
+      "post": {
+        "tags": [
+          "ManagedInstanceTdeCertificates"
+        ],
+        "description": "Creates a TDE certificate for a given server.",
+        "operationId": "ManagedInstanceTdeCertificates_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The requested TDE certificate to be created or updated.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TdeCertificate"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully created the TDE certificate."
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 MissingPrivateBlob - The private blob is missing.\n\n * 400 InvalidPrivateBlobOrPassword - Invalid private blob or password specified.\n\n * 400 InvalidResourceRequestBody - The resource or resource properties in the request body is empty or invalid.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ResourceNotFound - The requested resource was not found."
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Upload a TDE certificate": {
+            "$ref": "./examples/ManagedInstanceTdeCertificate.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "TdeCertificateProperties": {
+      "description": "Properties of a TDE certificate.",
+      "required": [
+        "privateBlob"
+      ],
+      "type": "object",
+      "properties": {
+        "certificateName": {
+          "description": "The name of the certificate",

```

Comment:
This property is readonly but the API is write-only. So this property is basically unused. Can you remove it?

Reply:
Thanks for review, I have addressed this comment in the latest commit.